src/views/product.js

```diff
@@ -19,7 +19,7 @@ export default class ProductView extends View {
    * check size of image until it is resolved, then set height of iframe.
    */
   resizeUntilLoaded() {
-    if (!this.iframe || !this.component.model.selectedVariantImage) {
+    if (!this.iframe || !this.component.currentImage) {
```

Comment:
`!(this.iframe && this.component.currentImage)` may read more clear? To me, this version looks like `neither A nor B`, as opposed to `not A or not B` which is really confusing.